lapce-core/Cargo.toml

```diff
@@ -21,5 +21,6 @@ tree-sitter-php = { git = "https://github.com/tree-sitter/tree-sitter-php.git",
 tree-sitter-c = "0.20.1"
 tree-sitter-cpp = "0.20.0"
 tree-sitter-json = "0.19.0"
+tree-sitter-markdown = { git = "https://github.com/MDeiml/tree-sitter-markdown.git", version = "0.0.1" }
```

Comment:
Despite not being a part of the official tree-sitter github, this is also used in neovim's syntax highlighting so I thought it adequate.